keyboards/star75/config.h

```diff
@@ -0,0 +1,66 @@
+/* 
+Copyright 2022 Horns Lyn (@hornslyn)
+SPDX-License-Identifier: GPL-2.0-or-later */
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4B35
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    DXWS
+#define PRODUCT         Star75
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 15
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { B7, D0, D1, D2, D3, D5 }
+#define MATRIX_COL_PINS { D4, D6, D7, B4, B5, B6, C6, C7, F7, F6, F5, F4, F1, F0, E6 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* number of backlight levels */
+#ifdef BACKLIGHT_PIN
+#define BACKLIGHT_LEVELS 3
+#endif
+
+/* encoder definitions */
+#ifdef ENCODER_ENABLE
+#define ENCODERS_PAD_A { B2 }
+#define ENCODERS_PAD_B { B1 }
+#define ENCODER_RESOLUTION 4
```

Comment:
Ideally, shouldn't be defined at the kb level, since it's very dependent on the hardware installed.  Also, 4 is default... so no need to set it, actually. 
```suggestion
```